DSCResources/MSFT_xADRecycleBin/Tests/ResourceDesignerTests.ps1

```diff
@@ -0,0 +1,2 @@
+Test-xDscResource xADRecycleBin -Verbose
+Test-xDscSchema -Path "C:\Program Files\WindowsPowerShell\Modules\xActiveDirectory\DSCResources\MSFT_xADRecycleBin\MSFT_xADRecycleBin.schema.mof" -Verbose
```

Comment:
Do you mean like this:
$Env:ProgramFiles\WindowsPowerShell\Modules\xActiveDirectory\DSCResources\MSFT_xADRecycleBin\MSFT...
or like this:
.\xActiveDirectory\DSCResources\MSFT_xADRecycleBin\MSFT_xADRecycleBin.schema.mof